config/config.go

```diff
@@ -9,22 +9,23 @@ import (
 
 // Configuration structure which hold information for configuring the import API
 type Configuration struct {
-	BindAddr                 string        `envconfig:"BIND_ADDR"`
-	KafkaAddr                []string      `envconfig:"KAFKA_ADDR"                       json:"-"`
-	AuditEventsTopic         string        `envconfig:"AUDIT_EVENTS_TOPIC"`
-	GenerateDownloadsTopic   string        `envconfig:"GENERATE_DOWNLOADS_TOPIC"`
-	CodeListAPIURL           string        `envconfig:"CODE_LIST_API_URL"`
-	DatasetAPIURL            string        `envconfig:"DATASET_API_URL"`
-	WebsiteURL               string        `envconfig:"WEBSITE_URL"`
-	ZebedeeURL               string        `envconfig:"ZEBEDEE_URL"`
-	DownloadServiceSecretKey string        `envconfig:"DOWNLOAD_SERVICE_SECRET_KEY"      json:"-"`
-	ServiceAuthToken         string        `envconfig:"SERVICE_AUTH_TOKEN"               json:"-"`
-	GracefulShutdownTimeout  time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
-	HealthCheckInterval      time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
-	EnablePrivateEnpoints    bool          `envconfig:"ENABLE_PRIVATE_ENDPOINTS"`
-	EnableDetachDataset      bool          `envconfig:"ENABLE_DETACH_DATASET"`
-	EnablePermissionsAuth    bool          `envconfig:"ENABLE_PERMISSIONS_AUTH"`
-	MongoConfig              MongoConfig
+	BindAddr                    string        `envconfig:"BIND_ADDR"`
```

Comment:
Might be useful for us to alphabetically sort these. Would reduce the amount of time a person has to spend to find an item in the list of config options.

Reply:
Im not a fan of doing this as it's very easy for it to become unsorted again as its not picked up for linting or complaining. IMO it's not worth the effort.